syne_tune/optimizer/schedulers/hyperband.py

```diff
@@ -445,8 +455,21 @@ def does_pause_resume(self) -> bool:
 
     @property
     def rung_levels(self) -> List[int]:
+        """
+        Note that all entries of ``rung_levels`` are smaller than ``max_t`` (or
```

Comment:
Previously, `rung_levels` sometimes ended on `max_t`, somtimes not, led to problems.